.vsts-ci.yml

```diff
@@ -107,19 +113,28 @@ jobs:
             _BuildConfig: Release
             _SignType: none
             _DotNetPublishToBlobFeed : false
+      variables:
+      - name: _RID
+        value: osx-x64
       steps:
       - checkout: self
         clean: true
       - script: eng/common/cibuild.sh
           --configuration $(_BuildConfig)
           --prepareMachine
-          /p:RID=osx-x64
+          /p:RuntimeIdentifier=$(_RID)
         displayName: Build
+      - task: CopyFiles@1
+        condition: and(eq(variables['system.pullrequest.isfork'], false), eq(variables['_BuildConfig'], 'Release'))
+        inputs:
+          sourceFolder: 'artifacts/bin/dotnet-core-uninstall/$(_BuildConfig)/netcoreapp3.0/$(_RID)/publish/'
+          contents: dotnet-core-uninstall
       - task: PublishBuildArtifacts@1
-        displayName: Upload package artifacts
         condition: and(eq(variables['system.pullrequest.isfork'], false), eq(variables['_BuildConfig'], 'Release'))
         inputs:
-          pathtoPublish: artifacts/bin/
-          artifactName: artifacts
-          artifactType: Container
+          pathtoPublish: '$(Build.ArtifactStagingDirectory)'
```

Comment:
where is this get published? a storage provided by Azure devops?

Reply:
Yes. I think this is a conventional approach to publish a build artifact. See https://docs.microsoft.com/en-us/azure/devops/pipelines/artifacts/build-artifacts?view=azure-devops&tabs=yaml.

Reply:
Even if we ship a `zip`, we still need to modify the artifact pipelines.

Reply:
We usually publish to a nuget feed or a blob storage we own. We haven't used Azure Devops for storage
@livarcocc make sure you are ok with this.

Reply:
We have a bunch of artifacts in azure devops. It is not where our customers consume them from, but the build does store stuff there.